test/conformance/helpers/tracing/traces.go

```diff
@@ -160,6 +160,10 @@ func GetTraceTree(trace []model.SpanModel) (*SpanTree, error) {
 		}
 	}
 
+	if len(roots) == 0 {
```

Comment:
Does it make sense to move this into getChildren? because right now getChildren is returning the wrong thing when 'current' is empty.

I do believe getChildren should validate what's passed into it. So adding this check in 2 places seems redundant, which is I why I recommended we move the check there.

Reply:
IIUC, moving the check into the `getChildren` block will break the terminal condition for a span which has no children, where the desired return value would be an empty SpanTree as opposed to an error. 

But thanks for the prompt, I some lines around for readability.

Reply:
makes sense.